control-plane/tests/test_service_persistence_assistant_advisory.py

```
@@ -1607,6 +1608,88 @@ def test_service_exposes_assistant_context_for_action_approvals_and_executions(
         )
         self.assertEqual(reconciliation_snapshot.reviewed_context, reviewed_context)
 
+    def test_service_includes_lifecycle_transition_history_in_generic_assistant_context(
+        self,
+    ) -> None:
+        store, service, promoted_case, evidence_id, reviewed_at = (
```

Ruff (0.15.10)

[warning] Unpacked variable `reviewed_at` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)